lib/oli/delivery/sections.ex

```diff
@@ -1020,6 +1025,39 @@ defmodule Oli.Delivery.Sections do
     |> Enum.map(fn e -> e.user end)
   end
 
+  @doc """
+  Returns the names of all instructors with :context_instructor role for the given section ids.
+
+  %{
+    section_id_1: [inst_1, inst_2],
+    ...
+    section_id_n: [inst_3]
+  }
+  """
+
+  def instructors_per_section(section_ids) do
+    instructor_context_role_id = ContextRoles.get_role(:context_instructor).id
+
+    query =
+      from(
+        e in Enrollment,
+        join: s in Section,
+        on: e.section_id == s.id,
+        join: ecr in EnrollmentContextRole,
+        on: e.id == ecr.enrollment_id,
+        where:
+          s.id in ^section_ids and s.status == :active and e.status == :enrolled and
```

Comment:
We shouldn't be including `s.status == :active` here.  If client code is passing in a list of section_ids, that client code might not be making the same assumptions that they only want results to return if the section is "active".